security/s2a/internal/crypter/common.go

```diff
@@ -0,0 +1,29 @@
+package crypter
+
+import "errors"
+
+const (
+	// GcmTagSize is the tag size - the difference in length between plaintext
+	// and ciphertext. From crypto/cipher/gcm.go in the Go crypto library.
```

Comment:
Please remove `the difference in length between plaintext and ciphertext`. (This will be true at the AEAD crypter layer but not at the record protocol layer. Since this is file is not AEAD crypter specific, I want to avoid confusion.)

Reply:
Please also add `the tag size in bytes`.